setup.py

```diff
@@ -53,7 +53,7 @@
     'Jinja2>=2.9.6,<2.11',
     'jsonschema[format]>=2.6.0,<2.7',
     'kubernetes>=6.0.0,<7.1',
-    'PyYAML>=3.12,<3.14',
+    'PyYAML>=4.2b4',
```

Comment:
Works well, but I see deprecation warnings when running `init` or `down`:

```
/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/reana_cluster/utils.py:31: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  cluster_spec = yaml.load(f.read())
/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/kubernetes/config/kube_config.py:460: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  config_dict=yaml.load(f),
/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/reana_cluster/backends/kubernetes/k8s.py:178: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  backend_conf_parameters = yaml.load(f.read())
/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/reana_cluster/backends/kubernetes/k8s.py:327: YAMLLoadWarning: calling yaml.load_all() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  for manifest in self.cluster_conf:
```

Can you please amend to provide the `Loader` to silence those warnings?

P.S. This is with PyYAML==5.1, we could upgrade also the requirement to '>=5.1' then. The version 4.2b4 does not emit warnings, but is not latest anymore...

Reply:
BTW we may also need to upgrade the `kubernetes` requirement to drop its upper limit in order to get rid of the warning there.  We could harmonise:
```console
$ grep 'kubernetes>' r*/setup.py
reana-cluster/setup.py:    'kubernetes>=6.0.0,<7.1',
reana-commons/setup.py:        'kubernetes>=6.0.0',
reana-job-controller/setup.py:    'kubernetes>=6.0.0',
reana-workflow-controller/setup.py:    'kubernetes>=6.0.0',
```

Reply:
I upgraded Kubernetes version in other open PRs. I am just missing r-j-c, since I have no open PR for this component.